tests/sod923.test.ts

```diff
@@ -0,0 +1,9 @@
+import { test, expect } from "bun:test"
+import { convertCircuitJsonToPcbSvg } from "circuit-to-svg"
+import { fp } from "../src/footprinter"
+
+test("sod923", () => {
+  const soup = fp.string("sod923").circuitJson()
```

Comment:
```suggestion
  const circuitJson = fp.string("sod923").circuitJson()
```

@techmannih Everytime!! 🤦‍♂️